rpmlint/lint.py

```diff
@@ -143,7 +149,9 @@ def _print_time_report(self):
                 if checked:
                     checked_files = checked
             print(f'    {check:32s} {duration:15.1f} {self._get_color_time_report_value(fraction)} {checked_files:>14}')
-        print(f'    {"TOTAL":32s} {total:15.1f} {100:17.1f} {total_checked_files:>14}\n')
+
+        total = format('Total', ':32s')
```

Comment:
This variable name conflicts with another one declared before, so it's overriding.

Reply:
Sorry for that, can I change newly created `total` variable to `totalf` meaning total formatted string?
That changes `totalf = format('Total', ':32s')`